GLAA.Domain/GLAAContextExtensions.cs

```diff
@@ -664,6 +664,96 @@ public static void Seed(this GLAAContext context, List<LicenceStatus> defaultSta
 
                 context.SaveChanges();
             }
+
+
+            //Public Register Test Licenses
+            if (!context.Licences.Any(x => x.ApplicationId == "ORG-1234"))
+            {
+                //completed application licenses
+                var completedLicences = new List<Licence>();
+
+                for (var i = 0; i < 50; i++)
+                {
+                    var licensedStatus =
+                        context.LicenceStatuses.FirstOrDefault(x => x.InternalStatus == "Licence issued – full");
+                    var country = string.Empty;
+                    var operatingCountries = new List<Country>();
+
+                    switch (i % 4)
+                    {
+                        case 0:
+                            country = "England";
+                            break;
+                        case 1:
+                            country = "Northern Ireland";
+                            operatingCountries.Add(context.Countries.FirstOrDefault(x => x.Name == "England"));
+                            break;
+                        case 2:
+                            country = "Wales";
+                            operatingCountries.Add(context.Countries.FirstOrDefault(x => x.Name == "England"));
```

Comment:
England?

Reply:
This probably all needs to change anyway as per the new country names.